GVFS/GVFS.Platform.Mac/MacPlatform.Shared.cs

```diff
@@ -15,7 +17,19 @@ public static bool IsElevatedImplementation()
 
         public static bool IsProcessActiveImplementation(int processId)
         {
-            // TODO(Mac): Implement proper check
+            try
+            {
+                Process process = Process.GetProcessById(processId);
+            }
+            catch (ArgumentException)
+            {
+                return false;
+            }
+            catch (InvalidOperationException)

```

Comment:
Minor: On the Windows platform we [don't catch](https://github.com/Microsoft/VFSForGit/blob/6dca004e2361927f96f2ebdbff3c351b4695bd38/GVFS/GVFS.Platform.Windows/WindowsPlatform.Shared.cs#L61) `InvalidOperationException`, and I am guessing because this appears to be [a bug in the documentation](https://stackoverflow.com/questions/52519030/net-getprocessbyid-when-is-invalidoperationexception-thrown).

As best I can tell this won't get thrown in [the .NET Core version on Mac](https://github.com/dotnet/corefx/blob/71122bb4921e0239ea679dd275d53695b94e52f4/src/System.Diagnostics.Process/src/System/Diagnostics/Process.cs#L998) either, and so I don't think we need to catch it here.

Reply:
(If you'd prefer to match the docs I recommend updating the Windows code to match instead)

Reply:
removed the un-necessary catch